test/extended/setup.sh

```diff
@@ -71,7 +75,9 @@ function os::test::extended::setup {
     # If the current system has the XFS volume dir mount point we configure
     # in the test images, assume to use it which will allow the local storage
     # quota tests to pass.
-    if [ -d "/mnt/openshift-xfs-vol-dir" ]; then
+    LOCAL_STORAGE_QUOTA=""
+    if [[ -d "/mnt/openshift-xfs-vol-dir" && $(lvs | grep -q "xfs") ]]; then
```

Comment:
We could:
```sh
if lvdisplay /dev/docker-vg/openshift-xfs-vol-dir; then

fi
```

I think that's more direct. However, I think we might need root privilege to do it